web-stories.php

```diff
@@ -60,15 +65,141 @@
 	define( 'WEBSTORIES_DEV_MODE', false );
 }
 
+// Load Compatibility class the old fashioned way.
+if ( ! class_exists( 'Web_Stories_Compatibility' ) ) {
+	require_once WEBSTORIES_PLUGIN_DIR_PATH . 'Web_Stories_Compatibility.php';
+}
+
+global $web_stories_compatibility;
```

Comment:
We should do `unset( $web_stories_compatibility )` at the end.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5212/commits/79feede30971484f1f72f7c2489605dd29401eed